### DISCUSSION TITLE
Thoughts on the Copilot costs?

### DISCUSSION THREAD

**Discussion by elsowiny** [community/community#19074] [Copilot Conversations]
Seems they're continuing access until August 22, then switching to a subscription model for $10 a month, or $100 month($20 dollar savings?)
Wondering what you guys think about this? I've tried other code suggestion plugins but feel this one has been very helpful.

**Comment by CharlVS** [comment]
I'm surprised it doesn't cost more. It pays for itself multiple times over for me with the hours saved. They could have easily charged $100+, and it would still be a bargain for most professionals, given the amount of saved time. That's what I was bracing myself for.

**Reply by rauzerfabiano** [reply]
> 

You don't take into account the situation of low-income people at all. The dollar for other countries is out of range.

Problem is just they trained their nice little tool on free and open source (and partially closed source) code and now make money from it.
And obviously the people who involuntarily contributed to the development of said tool don't get anything from the profit GitHub generates with it
